Cython/Compiler/Builtin.py

```diff
@@ -411,6 +426,8 @@ def _generate_divmod_function(scope, argument_types):
                                                   utility_code=UtilityCode.load("py_dict_clear", "Optimize.c")),
                                     BuiltinMethod("copy",   "T",   "T", "PyDict_Copy")]),
 
+    ("range",  "&PyRange_Type",    []),
```

Comment:
This can go in `types_that_construct_their_instance` I think (although right now there isn't a huge advantage to doing that)

Reply:
...and indeed it is